run_kraken.py

```diff
@@ -223,6 +268,7 @@ def main(cfg):
         global kubeconfig_path, wait_duration
         kubeconfig_path = config["kraken"].get("kubeconfig_path", "")
         chaos_scenarios = config["kraken"].get("chaos_scenarios", [])
+        litmus_version = config['kraken'].get("litmus_version", 'v1.9.0')
```

Comment:
s/v.1.9.0/v.1.9.1/
In line [18](https://github.com/openshift-scale/kraken/pull/33/files?diff=split&w=1#diff-38df760413887d0cc4ee5707919e6390456a863d326f6080644cc3d9c72e4bbaR18) you are applying version v.1.9.1